### PR TITLE
Support starting / stopping processes by group.

### PR DESCRIPTION
**Pull request by Sander-Toonen**
This PR will add the ability to start/ stop processes in a specific group and get only processes in specific groups. I will soon send a PR that will integrate this functionality in the SupervisorBundle as well.

### CODE REVIEW

File: src/Supervisor/Supervisor.php [hunk 1]
```diff
@@ -100,16 +100,22 @@ public function getProcessByNameAndGroup($name, $group)
     /**
      * getProcesses
      *
+     * @param array $limitToGroups Only show processes in these process groups.
+     *
      * @return Process[]
      */
-    public function getProcesses()
+    public function getProcesses($limitToGroups = array())
```
Comment (blaugueux) [inline src/Supervisor/Supervisor.php]: Can you please change variable name according your commit in SupervisorBundle?

**Comment by Sander-Toonen** [comment]
By the way, I didn't use the short array syntax here since you target PHP 5.3.0 with this library.

**Comment by blaugueux** [comment]
Right, but as PHP 5.3 is not supported anymore I must target 5.6. I'll do that ASAP.

**Comment by blaugueux** [comment]
Thanks for your work @Sander-Toonen 👍

**Pull request merged**